doc/release/1.9.0-notes.rst

```diff
@@ -70,60 +145,806 @@ New features
+here (to be linked).
+
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions to
+data.
+
+- The methods ``"pearson"`` and ``"tippet"`` from `scipy.stats.combine_pvalues`
+have been fixed to return the correct p-values, resolving
+[#15373](https://github.com/scipy/scipy/issues/15373). In addition, the
+documentation for `scipy.stats.combine_pvalues` has been expanded and improved.
+
+- Unlike other reduction functions, ``stats.mode`` didn't consume the axis
+being operated on and failed for negative axis inputs. Both the bugs have been
+fixed. Note that ``stats.mode`` will now consume the input axis and return an
+ndarray with the ``axis`` dimension removed.
+
+- Replaced implementation of `scipy.stats.ncf` with the implementation from
+Boost.
+
+- Add a `bits` parameter to `scipy.stats.qmc.Sobol`. It allows to use from 0
+to 64 bits to compute the sequence. Default is ``None`` which corresponds to
+30 for backward compatibility. Using a higher value allow to sample more
+points. Note: it does not correspond to the output type.
+
+- Add a `integers` method to `scipy.stats.qmc.QMCEngine`. It allows sampling
+integers using any QMC sampler.
+
+- improved the fit speed and accuracy of ``stats.pareto``
+
+- added ``qrvs`` method to ``NumericalInversePolynomial`` to match the
+situation for ``NumericalInverseHermite``
+
+- faster random variate generation for ``gennorm``
+
+- ``lloyd_centroidal_voronoi_tessellation`` has been added to allow improved
+sample distributions via iterative application of Voronoi diagrams and
+centering operations
 
 Other
 -----
 
+A draft developer CLI is available for SciPy, leveraging the ``doit``,
+``click`` and ``rich-click`` tools. For more details, see
+[gh-15959](https://github.com/scipy/scipy/pull/15959).
 
 
 
 *******************
 Deprecated features
 *******************
 
-`scipy.linalg` deprecations
-===========================
-
-
-`scipy.spatial` deprecations
-============================
-
-
+- Due to collision with the shape parameter ``n`` of several distributions,
+use of the distribution ``moment`` method with keyword argument ``n`` is
+deprecated. Keyword ``n`` is replaced with keyword ``order``. 
+- Similarly, use of the distribution ``interval`` method with keyword arguments
+``alpha`` is deprecated. Keyword ``alpha`` is replaced with keyword
+``confidence``.
+- The ``'simplex'``, ``'revised simplex'``, and ``'interior-point'`` methods
+of `scipy.optimize.linprog` are deprecated. Methods ``highs``, ``highs-ds``,
+or ``highs-ipm`` should be used in new code.
+- Support for non-numeric arrays has been deprecated from ``stats.mode``.
+``pandas.DataFrame.mode`` can be used instead.
+- The ``extradoc`` keyword of ``rv_continuous``, ``rv_discrete`` and
+``rv_sample`` now raises a DeprecationWarning, after having been deprecated in
+documentation only for a long time already.
+
+********************
+Expired Deprecations
```

Comment:
@h-vetinari @j-bowhay FYI if there are other entries to add, please add them here as suggestion so that it's easier for Tyler.

Reply:
I've tried to keep the wiki up-to-date with the issues in https://github.com/scipy/scipy/issues/15765 as they were closed. To the best of my knowledge, the list as below is complete.

Reply:
Thanks for confirming.